### DISCUSSION TITLE
API function to switch to fullscreen?

### DISCUSSION THREAD

**Discussion by RandomUnit** [nesbox/TIC-80#1512] [Ideas]
Is there anything like that? Exported executable always starts in windowed mode and under some circumstances it could be impossible for user to press F11 to switch it to fullscreen. I think it would be great if we could switch states through an API function so that player would for example tap an icon on the game's starting screen thus switching the game to fullscreen and back. Or maybe at least there are some workarounds like an argument to set the app state to fullscreen by default when exporting?
Sorry for my messy english, hope I've managed to express myself accessibly enough.

**Comment by nesbox** [comment]
Hi,
We are not going to add such API, sorry, but we could add `GO TO FULLSCREEN` item to the game menu.

**Reply by RandomUnit** [reply]
I guess that should do the trick. Should I make a pull request for that?

**Reply by nesbox** [reply]
Sure, if you want :)

**Reply by RandomUnit** [reply]
I think I'll have to leave it up to you because I have no idea how everything is working here, sorry(
